cansat-stm32f446/src/main.rs

```diff
@@ -4,12 +4,19 @@
 
 use cortex_m_rt::entry;
 use panic_halt as _;
-use stm32f4xx_hal as _;
+use stm32f4xx_hal as hal;
 use cansat::rtt_target::{rtt_init_print, rprintln};
 
+mod blink;
+
 #[entry]
 fn main() -> ! {
     rtt_init_print!();
     rprintln!("Hello, world!");
-    loop {}
+    
+    loop {
+        blink::blink_loop();
+        //will never exit function above
+    }
```

Comment:
If you apply my suggestions in the `blink_loop` fn, then this loop won't be needed
```suggestion
    blink::blink_loop();
```